monkey/monkey_island/cc/ui/src/components/report-components/security/issues/TunnelIssue.js

```diff
@@ -18,17 +18,27 @@ export function getAllTunnels(agents, machines) {
     if (!islandIPs.includes(agent.cc_server.ip)) {
       let agentMachine = getMachineByAgent(agent, machines);
       if (agentMachine !== null) {
-        let agentMachineHostname = getMachineHostname(agentMachine)
+        let agentMachineInfo = {
+          'id': agentMachine.id,
+          'ip': getMachineIPs(agentMachine)[0],
+          'hostname': getMachineHostname(agentMachine)
+        };
+
+        let agentTunnelMachineInfo = {
+          'id': -1,
+          'ip': agent.cc_server.ip,
+          'hostname': agent.cc_server.ip
+        };
 
-        let agentTunnelMachineHostname = agent.cc_server.ip;
         let agentTunnelMachine = getMachineFromIP(agent.cc_server.ip, machines)
         if (agentTunnelMachine !== null) {
```

Comment:
We'll run into issues if this happens. For example the `id` could be the same for multiple machines, so multiple components will have the same key, which will result in an error.

Reply:
When would the machine ID be the same for multiple machines?